packages/augur-test/src/tests/connector/ww-connector.test.ts

```diff
@@ -34,9 +34,6 @@
 // jest.mock("@augurproject/sdk/build/state/index", () => {
 //   return {
 //     __esModule: true,
-//     buildAPI: () => {
-//       // return new API(john.augur, db);
-//     },
```

Comment:
Delete the whole file maybe?

Reply:
oh heh the whole test is commented out....

I think we should fix it.